Healthcare-Hospital-Management-System/Healthcare-Hospital-Management-System/obj/Debug/net8.0/Healthcare-Hospital-Management-System.AssemblyInfo.cs

```diff
@@ -13,7 +13,7 @@
 [assembly: System.Reflection.AssemblyCompanyAttribute("Healthcare-Hospital-Management-System")]
```

Comment:
Required: remove bin and obj  folder, add them to gitignore, delete from repository